shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/stopDetailsPage/StopData.kt

```diff
@@ -0,0 +1,13 @@
+package com.mbta.tid.mbta_app.model.stopDetailsPage
+
+import com.mbta.tid.mbta_app.model.response.PredictionsByStopJoinResponse
+import com.mbta.tid.mbta_app.model.response.ScheduleResponse
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class StopData(
```

Comment:
Added in shared code in case we want to unite it with the swift representation. Schedules are a required field in the swift side, but it was easier to make it optional on the android side & let the predictions & schedules load independently with the way we are using coroutines.